website/blog/2020-05-18-rn4mupdadates.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "React Native for macOS"
+author: Kiki Saintonge
+authorURL: http://twitter.com/KikiSaintonge
+description: "We're on React Native core version 0.62 with new community modules, native module support across multiple platforms, and of course our official public release of React Native for macOS."
+is_blog: true
+published_date: May 17, 2020
+---
+
+We're on React Native core version 0.62 with new community modules, native module support across multiple platforms, and of course our official public release of React Native for macOS.
+
+<!--truncate-->
+
+## Version 0.62-preview Released
+
+As promised in our previous milestone post, we're chugging along and keeping as up-to-date with the latest React Native core version as we can. That means for this latest milestone release we are officially announcing our [0.62-preview of React Native for Windows](https://www.npmjs.com/package/react-native-windows/v/0.62.0-preview.6)!
+
+As usual this means that our getting started guides and general docs will also be updated to reflect these changes.
+
+## React Native for macOS
+
+![react-native-mac](assets/eloy_rn4m_preview_full.png)
+
+One of our biggest announcements recently is our addition and full support of the React Native for macOS extension! We are super excited to be developing React Native support for macOS along side our Windows effort.
+
+Although still a fork, we have an official [npm package out for React Native for macOS](https://www.npmjs.com/package/react-native-macos) and a [repo available](https://github.com/microsoft/react-native-macos) to the public to follow and contribute to - we hope to see some really cool React Native apps out there in the future that can now truly run everywhere.
+
+## New Module Support!
+
+We've begun addressing the Windows community module support gap - starting with two contributions from us! The [**WebView**](https://github.com/react-native-community/react-native-webview) and [**Camera**](https://github.com/react-native-community/react-native-camera) modules.
+
+On top of those initial two, we're looking at a whole bunch of other ones we know are awesome and essential to have when building a React Native app in general. Check out the full list of what we're tracking and planning to tackle next on our [GitHub Project Board](https://github.com/microsoft/react-native-windows/projects/23) for modules.
+
+We're also introducing release stages!
+
+### WebView
+
+![react-native-webview-module](assets/final_demo_gif.gif)
+
+The WebView module sets you up with the barebones web hosting tech that's available nativly on the device you're targetting - as a bonus it works with macOS as well!

```

Comment:
One more thing I would highlight as part of WebView support is our contribution to add CI to this repo for all platforms. Here is how I would word it- "As part of adding contributions to these community modules, we want to help bootstrap robust test infrastructure and enhance reliability for consumers of these modules. We have started this process by adding CI to `react-native-webview` repo. Stay tuned for more!"

Reply:
added!